portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,12 +44,13 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    final Query allComments = new Query(COMMENT);
+    final Query allComments = new Query(COMMENT).addSort(DATE_FIELD, SortDirection.DESCENDING);
```

Comment:
I recommend moving `COMMENT` and your other comment-related constants to your `Comment` class as public, so you can reference them from `DataServlet` and `DeleteDataServlet`.